orc8r/cloud/go/services/metricsd/servicers/southbound/servicer.go

```diff
@@ -15,9 +15,9 @@ package southbound
 
 import (
 	"context"
+	"errors"
```

Comment:
The cases in which std lib "errors" is imported without obvious use in the diff, are due to replacing errors.New from pkg/errors by errors.New from std lib, I assume?

Reply:
Yes, exactly. In the same file you should find a removal of github.com/pkg/errors